packages/mobile/android/app/src/main/java/com/zbaymobile/TorService.kt

```diff
@@ -58,10 +58,10 @@ class TorService: Service() {
         if(runTorCommand(torBinary, torrcCustom)) {
             var authCookie: String = ""
 
-            // val cookie = File(dataDirectory, "control_auth_cookie")
-            // if(cookie.exists()) {
-            //     authCookie = cookie.readBytes().toHex()
-            // }
+            val cookie = File(dataDirectory, "control_auth_cookie")
```

Comment:
do we have any tests for this control port auth stuff? 

if somebody changes something and it breaks, will it get caught by our tests?